src/node/delay.rs

```diff
@@ -117,7 +117,7 @@ impl AudioNode for DelayNode {
         output: u32,
         input: u32,
     ) -> &'a dyn AudioNode {
-        if self.context() != dest.context() {
+        if self.registration() != dest.registration() {
```

Comment:
This is very incorrect

Reply:
I stand corrected, it is very misleading.  I don't think you should override the equals method on this  struct

Reply:
This is not necessary if you don't want to hide BaseAudioContext. I can revert to context().